RHEL6_7/system/perl/check

```diff
@@ -6,9 +6,26 @@
 #END GENERATED SECTION
 
 FOREIGN_FOUND=0
-function log_file() {
-        printf '%s\n' "$@" >> "$SOLUTION_FILE" || exit_error
-        FOREIGN_FOUND=1
+log_file() {
+    printf '%s\n' "$@" >> "$SOLUTION_FILE" || exit_error
+    FOREIGN_FOUND=1
+}
+is_module_rh() {
+if [ $# -ne 1 ]; then
+     return 1
+fi
+RPM_NAME=$(rpm -qf --qf '%{NAME}\n' "$1")
+echo "$RPM_NAME" | grep "is not" >> /dev/null
+if [ $? -eq 0 ]; then
+    log_slight_risk "The $1 Perl module is not handled by any package."
+    log_file "$1"
+else
+    grep "^$RPM_NAME[[:space:]]"  $VALUE_RPM_RHSIGNED > /dev/null
```

Comment:
you should not read this file directly. For that purpose you have function `is_dist_native` which you should use.

Reply:
There was originally used that function. I have changed it FROM that function because it didn't serve well my purpose :-)  If you look at the `is_module_rh` it is partially based on `is_dist_native`, but tweaked a little bit. Overall it is just read operation ( it doesn't modify anything) and in this way the execution time of the module is cut shorter by 50 % which was a purpose of my fix.

Reply:
as was discussed, it will be used mentioned API function.